frontend/apps/web/components/site-header/AccountStatusHandler.tsx

```diff
@@ -35,3 +50,65 @@ export function AccountStatusHandler(props: Props) {
     </div>
   );
 }
+
+interface TrialCountdownProps {
+  trialEndDate: number;
+  isAccountStatusValidResp: IsAccountStatusValidResponse | undefined;
```

Comment:
might be worth simplifying these props to just pass in an `isExpired` and do the calculation outside of this function. make it less smart.